DigitalLearningSolutions.Web/appsettings.Development.json

```diff
@@ -1,6 +1,9 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=localhost;Initial Catalog=mbdbx101_uar;Integrated Security=True;"
+    //"DefaultConnection": "Data Source=localhost;Initial Catalog=mbdbx101;Integrated Security=True;"
+    "DefaultConnection": "Data Source=10.0.1.81;Initial Catalog=DLS_TEST;Persist Security Info=True;User ID=test_user;Password=mFZ5)ejyS}Z?VytJ;"
+    // "DefaultConnection": "Data Source=L6494;Initial Catalog=mbdbx101_uar;Persist Security Info=True;User ID=sa;Password=Password123;"

```

Comment:
Please don't change the connection string. They are for your local testing only. If this change gets merged into the master, it will have a huge impact on the whole system. Please be careful and understand how/when to make use of the UAT database.